backend/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM maven:3.8.5-openjdk-11
 WORKDIR /backend
 COPY . .
-RUN mvn clean install -Dmaven.test.skip=true
-CMD mvn spring-boot:run
\ No newline at end of file
+RUN mvn clean install -Pdocker
+CMD mvn spring-boot:run -Pdocker
```

Comment:
I don't believe you need to add profile when running spring app

Reply:
Since I've set <code>local</code> as the default Maven profile, without <code>-Pdocker</code> Maven injects <code>local</code> as the default Spring profile into <code>application.properties</code>.

Reply:
Well then don't set `local` as default one :)